i3df/src/bin/dump.rs

```diff
@@ -31,21 +30,10 @@ enum Output {
 fn main() -> Result<(), Box<dyn std::error::Error>> {
     let options = Options::from_args();
 
-    let root_file = options.root_file;
     let sector_file = options.sector_file;
 
-    let root_reader = BufReader::new(File::open(root_file)?);
     let sector_reader = BufReader::new(File::open(sector_file)?);
-    let root_sector = i3df::parse_root_sector(root_reader)?;
-    let attributes = match root_sector.header.attributes {
-        Some(x) => x,
-        None => {
-            return Err(Box::new(i3df::error::Error::new(
-                "Attributes missing on root sector",
-            )))
-        }
-    };
-    let raw_sector = i3df::parse_sector(&attributes, sector_reader)?;
+    let raw_sector = i3df::parse_root_sector(sector_reader)?;
```

Comment:
Let us rename the functions in i3df while we are at it. I know I will forget to do so otherwise :stuck_out_tongue: